label_studio/io_storages/base_models.py

```diff
@@ -515,6 +523,7 @@ def sync(self):
                 self.info_set_queued()
                 import_sync_background(self.__class__, self.id)
             except Exception:
+                logger.debug(f'Storage {self} failed', exc_info=True)
```

Comment:
```suggestion
                # needed to facilitate debugging storage-related testcases, since otherwise no exception is logged
                logger.debug(f'Storage {self} failed', exc_info=True)
```

Reply:
This is a rare case where I'm in favor of a comment - the reason it's here is fairly subtle and not at all obvious from the context